src/components/ChangePlan/CheckoutProcess.js

```diff
@@ -0,0 +1,361 @@
+import { Loading } from '../Loading/Loading';
+import {
+  FieldGroup,
+  FieldItem,
+  FormWithCaptcha,
+  InputFieldItem,
+  PhoneFieldItem,
+  SelectFieldItem,
+  SubmitButton,
+} from '../form-helpers/form-helpers';
+import { Helmet } from 'react-helmet';
+
+const ContactInformation = ({
+  contactInformation,
+  countries,
+  steps,
+  setActiveStep,
+  handleSaveAndContinue,
+}) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const fieldNames = {
+    firstname: 'firstname',
+    lastname: 'lastname',
+    address: 'address',
+    city: 'city',
+    province: 'province',
+    country: 'country',
+    zipcode: 'zipcode',
+    phone: 'phone',
+    company: 'company',
+    industry: 'industry',
+  };
+
+  const getFormInitialValues = () => {
+    const initialValues = Object.keys(fieldNames).reduce(
+      (accumulator, currentValue) => ({
+        ...accumulator,
+        [currentValue]: '',
+      }),
+      {},
+    );
+
+    if (contactInformation !== null) {
+      initialValues[fieldNames.firstname] = contactInformation.firstname;
+      initialValues[fieldNames.lastname] = contactInformation.lastname;
+      initialValues[fieldNames.address] = contactInformation.address;
+      initialValues[fieldNames.city] = contactInformation.city;
+      initialValues[fieldNames.province] = contactInformation.province;
+      initialValues[fieldNames.country] = contactInformation.countryId;
+      initialValues[fieldNames.zipcode] = contactInformation.zipCode;
+      initialValues[fieldNames.phone] = contactInformation.phoneNumber;
+      initialValues[fieldNames.company] = contactInformation.company;
+      initialValues[fieldNames.industry] = contactInformation.industry;
+    }
+
+    return initialValues;
+  };
+
+  const [formSubmitted, setFormSubmitted] = useState(false);
+  const [isCompletedStep, setCompletedStep] = useState(false);
+
+  const submitContactInformationForm = async (values, { setSubmitting }) => {
+    setFormSubmitted(false);
+    try {
+      const result = await handleSaveAndContinue(values);
+      if (result.success) {
+        setFormSubmitted(true);
+        setCompletedStep(true);
+      }
+    } finally {
+      setTimeout(() => setFormSubmitted(false), 3000);
+    }
+  };
+
+  const editContactInformation = () => {
+    setActiveStep('contact-information');
+  };
+
+  /* TODO: These styles will be removed with the correct classes. Currently Gus is working with the layout. */
+  const containerStyle = {
+    paddingLeft: '20px',
+    paddingTop: '20px',
+    paddingRight: '20px',
+    border: '2px solid #eaeaea',
+    borderRadius: '3px',
+  };
+  const containerMessageSucces = { padding: '0px' };
+  const flextStartSyle = { justifyContent: 'flex-start' };
+
+  return (
+    <div className="dp-wrapper-form-plans contact-information" style={containerStyle}>
+      <div style={{ display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
I think we try not to use inline styles @cbernat , it's right ?

Reply:
this style is temporarybecause gus is working with the layout when the layout is ready I will remove all inline styles.

Reply:
I added a TODO in the code:  "TODO: These styles will be removed with the correct classes. Currently Gus is working with the layout."

Reply:
Yes it's temporal. And it has a TODO.